src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -985,6 +985,22 @@ def test_validate_params(self):
         assert "12345/8/1" in ws_inputs
         assert "12345/7/1" in ws_inputs
 
+    @mock.patch(CLIENTS_AM_SM, get_mock_client)
+    @mock.patch(CLIENTS_SM, get_mock_client)
+    def test_validate_params_wildcard_type(self):
```

Comment:
As above, add another test case.

This test module gets its app specs straight from the Narrative Method Store. So might as well just use the one that was causing problems.